src/main/kotlin/no/nav/k9/pleiepengersyktbarn/soknad/PleiepengerSyktBarnSoknadValidator.kt

```diff
@@ -22,21 +22,24 @@ class SoknadValidator : ConstraintValidator<ValidPleiepengerSyktBarnSoknad, Plei
             context: ConstraintValidatorContext?): Boolean {
         var valid = true
 
-        fun validerPeriode(periode: Periode?, prefix: String) : Boolean {
+        fun validerPeriode(periode: Periode?, prefix: String?) : Boolean {
+
+            val prefixPeriode = if (prefix == null) "periode" else "$prefix.periode"
```

Comment:
La inn dette for å tilrettelegge for feilmeldinger knyttet til den overordnede perioden, som ligger på roten av søknadsobjektet.